build-system/global-configs/client-side-experiments-config.json

```diff
@@ -23,6 +23,10 @@
     {
       "name": "story-load-inactive-outside-viewport",
       "percentage": 1
+    },
+    {
+      "name": "story-ad-page-outlink",
+      "percentage": 1
```

Comment:
Let's start with beta only. Also we need to make sure not to merge this before the changes are live (should be Wednesday).

Reply:
Ok, sounds good, I'm starting with Beta only.